src/test/java/uk/gov/hmcts/reform/bulkscan/services/BulkScanC100ServiceTest.java

```diff
@@ -42,114 +56,148 @@ class BulkScanC100ServiceTest {
     }
 
     @Test
-    void testC100CaseNoNumericErrorWhileDoingValidation() {
-        BulkScanValidationRequest bulkScanValidationRequest = BulkScanValidationRequest.builder().ocrdatafields(
-            TestDataUtil.getNumericErrorData()).build();
-        BulkScanValidationResponse res = bulkScanValidationService.validate(bulkScanValidationRequest);
-        assertEquals(Status.ERRORS, res.status);
-        assertTrue(res.getWarnings().items.contains(String.format(NUMERIC_MESSAGE, "case_no")));
+    @DisplayName("C100 transform Child live with respondent.")
+    void testC100TransformWithRespondent() throws IOException, JSONException {
+        BulkScanTransformationRequest bulkScanTransformationRequest = mapper.readValue(
+            readFileFrom(C100_TRANSFORM_REQUEST_PATH), BulkScanTransformationRequest.class);
+        bulkScanTransformationRequest.getOcrdatafields().stream()
+            .filter(eachField ->
+                        CHILD_LIVING_WITH_APPLICANT.equalsIgnoreCase(eachField.getName()))
+            .forEach(field -> field.setValue("false"));
+        bulkScanTransformationRequest.getOcrdatafields().stream()
+            .filter(eachField ->
+                        CHILD_LIVING_WITH_RESPONDENT.equalsIgnoreCase(eachField.getName()))
+            .forEach(field -> field.setValue("true"));
+        BulkScanTransformationResponse res = bulkScanValidationService.transform(bulkScanTransformationRequest);
+        assertEquals(
+            "Respondent",
+            res.getCaseCreationDetails().getCaseData().get(CHILD_LIVE_WITH_KEY));
     }
 
     @Test
-    void testTransform() {
-        BulkScanTransformationResponse bulkScanTransformationResponse =
-            bulkScanValidationService.transform(mock(BulkScanTransformationRequest.class));
-        Assertions.assertNotNull(bulkScanTransformationResponse);
+    @DisplayName("C100 transform Child live with others.")
+    void testC100TransformWithOthers() throws IOException, JSONException {
+        BulkScanTransformationRequest bulkScanTransformationRequest = mapper.readValue(
+            readFileFrom(C100_TRANSFORM_REQUEST_PATH), BulkScanTransformationRequest.class);
+        bulkScanTransformationRequest.getOcrdatafields().stream()
+            .filter(eachField ->
+                        CHILD_LIVING_WITH_APPLICANT.equalsIgnoreCase(eachField.getName()))
+            .forEach(field -> field.setValue("false"));
+        bulkScanTransformationRequest.getOcrdatafields().stream()
+            .filter(eachField ->
+                        CHILD_LIVING_WITH_OTHERS.equalsIgnoreCase(eachField.getName()))
+            .forEach(field -> field.setValue("true"));
+        BulkScanTransformationResponse res = bulkScanValidationService.transform(bulkScanTransformationRequest);
+        assertEquals(
+            "OtherPeople",
+            res.getCaseCreationDetails().getCaseData().get(CHILD_LIVE_WITH_KEY));
     }
 
```

Comment:
Should testC100ValidationErrorWithSicialAuthority be testC100ValidationErrorWithSocialAuthority

Reply:
done.